src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${not empty noResponsePanel.instructorStatus[email]}">
+                                     <td><i>${fn:escapeXml(noResponsePanel.teams[email])}</i></td>
+                            </c:if>
+                            <c:if test="${empty noResponsePanel.instructorStatus[email]}">
```

Comment:
Let's make this check slightly stronger by checking for more than emptiness, this should also check if the value is false if not empty. Likewise, for the condition right above this.